<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotDlg</class>
 <widget class="QDialog" name="PlotDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>41</x>
          <y>0</y>
          <width>601</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="hourButton">
           <property name="text">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="dayButton">
           <property name="text">
            <string>D</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="weekButton">
           <property name="text">
            <string>W</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="monthButton">
           <property name="text">
            <string>M</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="yearButton">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLCDNumber" name="totalLcd"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
